spacy/tests/lang/de/test_noun_chunks.py

```diff
@@ -0,0 +1,28 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import pytest
+from spacy.language import Language
+from spacy.vocab import Vocab
+from spacy.tokens import Doc
+
+
+@pytest.fixture
+def nlp():
+    return Language()
+
+
+def test_noun_chunks_is_parsed_de():
+    """Test that noun_chunks raises Value Error for 'de' language if Doc is not parsed. 
+    To check this test, we're constructing a Doc
+    with a new Vocab here and forcing is_parsed to 'False' 
+    to make sure the noun chunks don't run.
+    """
+    try:
+        doc = Doc(Vocab(), words=["Er", "lag", "auf", "seinem"])
+        doc.is_parsed = False
+        list(doc[0:3].noun_chunks)
+    except ValueError:
+        pass
+    else:
+        pytest.fail("Parsing did not catch a E029 ValueError for language 'de'")
```

Comment:
You can use the language-specific tokenizer that's already been initialized for use in the test suite and `pytest.raises` to detect the error:

```
def test_noun_chunks_is_parsed_de(de_tokenizer):
    """Test that noun_chunks raises Value Error for 'de' language if Doc is not parsed. 
    To check this test, we're constructing a Doc
    with a new Vocab here and forcing is_parsed to 'False' 
    to make sure the noun chunks don't run.
    """
    doc = de_tokenizer("Er lag auf seinem")
    doc.is_parsed = False
    with pytest.raises(ValueError):
        list(doc.noun_chunks)
```

There should be a tokenizer for each language called `lg_tokenizer` for language `lg`.

Reply:
Hi @adrianeboyd , Thanks for the review and suggestion. I will make the changes and update the PR accordingly. :)

Reply:
Hi @adrianeboyd , I have addressed the review comments, retested them and it is working fine with the changes. Can you please review and merge it? Thanks!

Reply:
Looks good! The only remaining thing would be to clean up the now unused imports and fixtures in the tests and then I think it will be ready to be merged.

Reply:
Sure @adrianeboyd, I will work on it and get back to you.